1678.goal-parser-interpretation.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * @param {string} command
+ * @return {string}
+ */
+
+/* URL of this problem
+    https://leetcode.com/problems/goal-parser-interpretation/
+*/
+
+var interpret = function(command) {
+    command = command.replaceAll("()", "o");
+    command = command.replaceAll("(al)", "al");
+    
+    return command;
```

Comment:
it is better not to change command parameter. It is called side effect.
```suggestion
    return command.replaceAll("()", "o").replaceAll("(al)", "al");
```